runtime/assembly/__tests__/inherent.spec.ts

```diff
@@ -1,14 +1,13 @@
-import { Inherent } from '../models';
+import { InherentData } from '../models';
 import { MockBuilder } from './mock-builder';
 import { MockResult } from './mock-result';
 
 describe("Inherent", () => {
     it("Should instanciate new Inherent class from the empty SCALE Encoded Array", () => {
-        const mock: MockResult<Inherent> = MockBuilder.getInherentMock();
-        const decodedData = Inherent.fromU8Array(mock.bytes);
-
-        expect(decodedData == mock.instance).toBeTruthy();
+        const mock: MockResult<InherentData> = MockBuilder.getInherentMock();
+        const decodedData = InherentData.fromU8Array(mock.bytes);
+        assert(decodedData.result == mock.instance, "Expected object is not equal to the newly instanciated object");
         __retain(changetype<usize>(mock));
         __retain(changetype<usize>(decodedData));
     })
-})
\ No newline at end of file
+})
```

Comment:
Once we have the assertion for the length of the `InherentMap`, we could add `throws` test that checks that if someone instantiates `InherentData` with keys that do not have 8 bytes length, it will throw